platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/admin/ConfigurationAdminMigratable.java

```diff
@@ -155,8 +160,12 @@ private boolean isAcceptedConfigurationAdminEntry(ImportMigrationConfigurationAd
     if (ACCEPTED_ENTRY_PIDS.contains(entry.getPid())) {
       return true;
     }
-    if (ACCEPTED_ENTRY_FACTORY_PIDS.contains(entry.getFactoryPid())) {
-      return true;
+    if (entry.getFactoryPid() != null) {
```

Comment:
✏️ You don't need this `null` check. If the factoryPid is `null`, then the next condition will return false since the `ACCEPTED_ENTRY_FACTORY_PIDS` doesn't have `null` as one of its items.

Reply:
Wouldn't we still need the null check for the other source configurations because it is an or?

Reply:
According to the javadoc for `StringUtils.containsIgnoreCase`, that method should handle `null`. Did you see something different?
https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html#containsIgnoreCase-java.lang.CharSequence-java.lang.CharSequence-

Reply:
I remember getting a nullException at one point here, but I can't remember at what state. I will try again with this updated version and report back.

Reply:
Updated